### PR TITLE
Remove postmark

### PR DESCRIPTION
**Pull request by esaborit4code**

### CODE REVIEW

File: config/environments/staging.rb [hunk 1]
```diff
@@ -1,13 +1,6 @@
 require_relative "production"
 
 Mail.register_interceptor(
-  RecipientInterceptor.new(ENV.fetch("EMAIL_RECIPIENTS"))
```
Comment (jguitar) [inline config/environments/staging.rb]: Why did you remove the `ENV.fetch`? Just curious.

Comment (esaborit4code) [reply config/environments/staging.rb]: I just copied the full block from another project.
In addition, it wasn't either specifying a default value for the fetch nor requiring an exception to be thrown, so it doesn't really matter.
